core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci

```diff
@@ -66,3 +66,4 @@ quay.io/ocs-dev/odf-console-ci-runner:current registry.ci.openshift.org/ci/odf-c
 quay.io/ocs-dev/odf-console-ci-runner:latest registry.ci.openshift.org/ci/odf-console-ci-runner:latest
 quay.io/strimzi-qe/strimzi-tests:release-0.29.x registry.ci.openshift.org/ci/strimzi-tests:release-0.29.x
 quay.io/strimzi-qe/strimzi-tests:release-0.32.x registry.ci.openshift.org/ci/strimzi-tests:release-0.32.x
+quay.io/redhat_msi/ocp-addons-operators-cli:latest registry.ci.openshift.org/ci/ocp-addons-operators-cli:latest
```

Comment:
In that separate PR I would also do the same for this image

`quay.io/modh/ods-ci:latest`